### PR TITLE
AWS csfle example

### PR DESCRIPTION
**Pull request by dangingera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +24,143 @@ We then start a consumer with the corresponding configurations to decrypt the fi
...
+
+After your Key has been created, navigate to AWS IAM and create an Access Key for the User that you granted permissions to in the previous step.
+
+![](images/aws_create_access_key.jpg)
+
+:warning: **Important:** Ensure you copy your Access Key ID and Secret (or download the csv file) :warning:
+
+## Update the Producer and Consumer Properties Files
+
+Update the `rule.executors._default_.param.access.key.id` and `rule.executors._default_.param.secret.access.key` properties for the Producer and Consumer with the Access Key and Secret you retrieved in the previous step.
```
Comment (pneff93) [inline README.md]: This part is a duplicate of the properties later on. Is it required or can be removed?

Comment (dangingera) [reply README.md]: good point, I will adjust

File: KafkaProducer/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+dependencies {
+    implementation(kotlin("stdlib"))
+    implementation("org.jetbrains.kotlin:kotlin-reflect:1.5.31")
+    implementation("org.apache.logging.log4j:log4j-api-kotlin:1.0.0")
+    implementation("org.apache.logging.log4j:log4j-core:2.12.0")
+    implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.12.0")
+    implementation("io.ktor:ktor-server-netty:1.2.2")
+    implementation("com.google.code.gson:gson:2.8.8")
+    implementation("io.confluent:kafka-avro-serializer:7.5.1")
+    implementation("io.confluent:kafka-schema-registry-client-encryption-gcp:7.5.1")
```
Comment (pneff93) [inline KafkaProducer/build.gradle.kts]: should be kafka-schema-registry-client-encryption-aws

Comment (dangingera) [reply KafkaProducer/build.gradle.kts]: done - forgot to commit

**Pull request merged**